### ISSUE TITLE
Where to change configuration settings

### ISSUE THREAD

**Issue by elgholm** [Open-Web-Analytics/Open-Web-Analytics#514]
Hi!

I'm trying to understand https://github.com/Open-Web-Analytics/Open-Web-Analytics/wiki/Configuration about where to change (set or read) configuration settings.

I understand that owa-config.php is read, but that file only contains the define-commands.

It's then stated that you can write
$this->set('base, 'mailer-from', 'owa@myinstance.com');
To set configuration directives. But. Where? In which file?
I guess it's not in the owa-config.php file? ($this?)

I rather not change default parameters in the modules, and would really like to understand how and where I'm supposed to find and/or set the configuration settings for my new installation.

I'm trying to figure out how the make the e-mailing to use our default smtp-server, which is correctly installed with TLS/SPF/DKIM/DMARC/Reverse-DNS etc..

tl;dr; how can I view all current configuration settings, and where to change them?

**Comment by Maaiins** [comment]
Hey elgholm,

you can use `$this->set('base, 'mailer-from', 'owa@myinstance.com');` in `owa-config.php`. Just put the line at the end of the file.

As you can see in #499 this is a todo which is going to be changed in a future release.

**Comment by elgholm** [comment]
Ok, that was.. weird. ;)
But perfect, and thanks for asking the question so fast! <3
Now I just have to figure out what settings for the smtp I can change, but I saw a bunch in the modules..settings.php, so I'll figure it out! Thx!
